### DISCUSSION TITLE
SO - VM running from External Hard Drive

### DISCUSSION THREAD

**Discussion by deancybersec** [Security-Onion-Solutions/securityonion#9286] [General]
Hi everybody!

Downloading the .iso now to try test the new version in my lab.

I wish to install SO Latest version: 2.3.182 in a VM and run it from VM Fusion on Mac (non M1) on an external harddrive non-bootable harddrive.

Any tricks or tips for success for this?

**Comment by dougburks** [comment]
Please make sure you are following the hardware requirements for whatever kind of installation you choose:
https://docs.securityonion.net/en/2.3/hardware.html

In particular, please note that we recommend SSD for best results.
